src/modules/addTask.js

```diff
@@ -0,0 +1,14 @@
+import { addTaskInput } from './elements.js';
+import createTask from './createTask.js';
+
+let dataTaskAdd = [];
+// add a new task
+const addTask = () => {
+  dataTaskAdd = JSON.parse(localStorage.getItem('dataTasks'));
+  const lengthList = dataTaskAdd.length + 1;
+  dataTaskAdd.push({ description: addTaskInput.value, completed: false, index: lengthList });
+  addTaskInput.value = '';
+  createTask(lengthList - 1, lengthList, dataTaskAdd);
+  localStorage.setItem('dataTasks', JSON.stringify(dataTaskAdd));
+};
```

Comment:
- [ ] After being able to solve the first error, another error popped up when I tried to add a `list item` so I will suggest you fix that as well :100: So users are able to add and remove items in your app :+1: 
- [ ] However, I modified this part of your code to look something like this below and afterward the app was working fine. So kindly fix that as well :
```javascript
const addTask = () => {
  const storedData = localStorage.getItem('dataTasks');
  if (storedData) {
    dataTaskAdd = JSON.parse(storedData);
  } else {
    dataTaskAdd = [];
  }
  ```
- Error Message
![image](https://github.com/ivangonzalez224/To-do-list/assets/102699958/25f2b9b7-6cb0-4285-a2e0-94fd90035110)